### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Arnab-Developer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,3 +12,5 @@ update 25-may-2024
 12-July-2024 8 hours 19 minutes AM
 
 This is a PR review with assignment test.
+
+update on 21 Aug 2024
```
Comment (arnab-jit1) [inline README.md:16]: comment 1

Comment (Arnab-Developer) [reply README.md:16]: I will do it.

Comment (arnab-jit1) [inline README.md:16]: c3

Comment (Arnab-Developer) [reply README.md:16]: will do

**Review by arnab-jit1 — APPROVED** [review]

**Pull request merged**